recipes/protobuf/all/conandata.yml

```diff
@@ -1,62 +1,99 @@
 sources:
+  "5.27.0":
+    url: "https://github.com/protocolbuffers/protobuf/archive/refs/tags/v5.27.0.tar.gz"
+    sha256: "1611a03e550c27c526bde1da544c94f7aa65c10687bbc8e570537dfa94069e1a"
+  "4.25.3":
+    url: "https://github.com/protocolbuffers/protobuf/archive/refs/tags/v4.25.0.tar.gz"
```

Comment:
Why the 4.25.0 tarball with the 4.25.3 version?

Reply:
Very well spotted!! I tested several versions and this was a leftover. Thanks for noticing, will fix ASAP